### PR TITLE
feat: homepage static design done

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Added:
Navbar
Homepage desgin

### CODE REVIEW

**Comment by mayuresh-tech** [comment]
Preview: https://jade-strudel-cd4753.netlify.app/

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              <div id="logo-container">
+                <div id="logo">
+                  <img
+                    id="main-logo"
+                    width="30px"
+                    height="30px"
+                    src="favicon.svg"
+                    alt="Logo"
+                  />
+                  <div>Zippy UI</div>
```
Comment (deekshithmd) [inline src/components/navbar/Navbar.jsx]: Check whether it needs to be changed...

Comment (mayuresh-tech) [reply src/components/navbar/Navbar.jsx]: Fixed in next commit

File: src/routes/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <div>
+      <Navbar />
+      <CategoriesGrid />
+      <Collection />
+      <Offers />
+    </div>
+  );
+}
+
+export default HomePage;
```
Comment (deekshithmd) [inline src/routes/HomePage.jsx:21]: Code looks good you can proceed to merge...Good work on homepage design

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**